eppo-sdk-test/dto/BanditReferencesTest.cs

```diff
@@ -0,0 +1,78 @@
+using eppo_sdk.dto;
+using static NUnit.Framework.Assert;
+
+namespace eppo_sdk_test.dto;
+
+
+public class BanditReferencesTest
+{
+    private BanditReferences banditRefs;
+    [SetUp]
+    public void SetUp()
+    {
+        banditRefs = new BanditReferences()
+        {
+            // Typical `BanditVariationDto` values where `variation` is duplicated across the VariationValue, VariationKey and BanditKey fields.
+            ["variation"] = new BanditReference(
```

Comment:
minor, but consider naming it `banditVariation` or `banditKey` or something to be more realistic and explicit